internal/field.go

```diff
@@ -83,15 +83,27 @@ func getFieldContent(ctx context.Context, doc *enigma.Doc, fieldName string, cou
 	}
 
 	var result []string
-	// Get hypercube layout
 
+	// If there are no datapages, it is (probably?) not a field.
+	if len(layout.ListObject.DataPages) == 0 {
+		errMsg := fmt.Sprintf("Error: No field by name '%s'", fieldName)
+		FatalError(errMsg)
+	}
+
+	// Get hypercube layout
 	for _, page := range layout.ListObject.DataPages {
 		for _, row := range page.Matrix {
 			for _, cell := range row {
+				text := cell.Text
```

Comment:
An ide is to show both the text and the num presentation🤔

Could be god in som cases e.g
" 30" != "30"

But this only shows the top soo maybe not that useful